doc/sphinx/source/tbb.rst

```diff
@@ -0,0 +1,42 @@
+==========
+TBB device
+==========
+
+The TBB device uses the Intel Threading Building Blocks open source (Apache 2.0)
+library for scheduling and presents an alternative to the pthread device.
+
+Contrary to the pthread device, the TBB device scheduling characteristics can be
+fine tuned with environment variables (see below).
+
+Building pocl with TBB
+----------------------
+
+1) Install prerequisites
+~~~~~~~~~~~~~~~~~~~~~~~~
+  The Intel Threading Building Blocks library must be available on your system.
+
+  pocl uses a `third party CMake module <https://github.com/justusc/FindTBB>`_
+  to locate the library on your system.
+
+2) Build pocl
+~~~~~~~~~~~~~
+  To enable the TBB device, add ``-DENABLE_TBB_DEVICE=1`` to your CMake
+  configuration command line.
+
+  If CMake has trouble locating the TBB library, try specifying
+  ``TBB_ROOT_DIR``, ``TBB_INCLUDE_DIR`` and ``TBB_LIBRARY``.
+  See the `related documentation <https://github.com/justusc/FindTBB>`_ for
+  further help.
+
+3) Configuration
+~~~~~~~~~~~~~~~~
+  Use ``POCL_DEVICES=tbb`` to select the TBB device.
+
+  Optionally, set the ``POCL_TBB_PARTITIONER`` environment variable to one of
```

Comment:
Is there always at most one TBB device which controls all CPUs? PoCL allows defining a list of devices via POCL_DEVICES and parameters per device id. What happens if there are more than one TBB devices? Error out?

Reply:
It is strongly recommended to use at most on TBB device. I added this to the [documentation](https://github.com/new2f7/pocl/commit/6efa2d06a76e0d9982fb693269069c011b56b55f#diff-fb6a8d163fc0c1e73f9d455ba2b8eccf979def2873d43e04d4a547f9d76f090aR38).